CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## XX.XX.XX - 20XX-XX-XX
 
+## 20.52.0 - 2024-10-07
+
+### CustomerSheet
+* [ADDED] `CustomerSheet` is now in GA. For more information see our docs [here](https://docs.stripe.com/elements/customer-sheet).
```

Comment:
should this say "is now generally available" instead of "in GA"? I'm not sure if the latter wording would be common outside of Stripe